src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -214,6 +214,22 @@ class CommandRequestImpl : public CommandImpl,
   mobile_apis::Result::eType GetMobileResultCode(
       const hmi_apis::Common_Result::eType& hmi_code) const;
 
+  /**
+   * @brief Checks result code from Mobile for single RPC
+   * @param result_code contains result code from Mobile response
+   * @return true if result code complies successful result codes,
+   * false otherwise.
+   */
+  static bool IsMobileResultSuccess(mobile_apis::Result::eType result_code);
+
+  /**
+   * @brief Checks result code from HMI for single RPC
+   * @param result_code contains result code from Mobile response
+   * @return true if result code complies successful result codes,
```

Comment:
same here

Reply:
@mked-luxoft fixed in 5db179eb26efd1a99c0178db3afb9d1e077cb383